Lib/pkgutil.py

```diff
@@ -465,6 +465,10 @@ def get_loader(module_or_name):
     If the named module is not already imported, its containing package
     (if any) is imported, in order to establish the package __path__.
     """
+    warnings.warn("`pkgutil.get_loader` is deprecated since Python 3.12; "
+                  "this function is slated for removal in Python 3.14, "
+                  "use `importlib.util.find_spec` instead",
+                  DeprecationWarning)
```

Comment:
Use `stacklevel=2` to make sure the warning or error trace shows the calling module ?